projects/aca-content/src/lib/components/bulk-actions-dropdown/bulk-actions-dropdown.component.html

```diff
@@ -17,7 +17,7 @@
       [value]="option.id"
       [title]="option.tooltip | translate"
       [attr.data-automation-id]="option.id"
-      (click)="runAction(option)"
+      (onSelectionChange)="runAction({event: $event, actionOption: option})"
```

Comment:
Why do we need this change? You're passing an event and then you only check if the option was selected

Reply:
`onSelectionChange` handles both Enter selection and click selection, but it fires twice, for selecting new item and unselecting previous and using $event allows to discern those two calls. I don't think `mat-option`'s keypress.enter can be overwritten.